src/AuthService.DatabaseMigrator/AuthService.DatabaseMigrator.csproj

```diff
@@ -18,6 +18,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\AuthService.BusinessLogic\AuthService.BusinessLogic.csproj" />
```

Comment:
Migrator should no have access to business layer directly. Only DataAccess should be here. Please remove project reference because it is useless.